Tribler/Core/Config/config.spec

```diff
@@ -25,6 +25,11 @@ ec_keypair_filename = string(default='')
 testnet_keypair_filename = string(default='')
 live_edges_enabled = boolean(default=True)
 
+[bootstrap]
+enabled = boolean(default=True)
+download_rate = integer(min=1, default=1000000)
```

Comment:
Could you rename this to `max_download_rate` (since that is essentially what it is)?